lib/features/account_setting_screen/widget/account_setting_mobile.dart

```diff
@@ -13,11 +13,11 @@ class AccountSettingsMobileMode extends StatelessWidget
         children: [
           verticalSpace(33),
           IconButton(
-            icon: Icon(Icons.arrow_back, color: Colors.black),
-            onPressed: () => Navigator.pop(context),
-          ),
+              onPressed: () => GoRouter.of(context).pop(),
+              icon: SvgPicture.asset(AppIcons.arrowBack),
+            ),
           verticalSpace(24),
-          Text("ACCOUNT SETTINGS",
+          Text("ACCOUNT SETTINGS".tr(),
```

Comment:
you can't just do this only, see en.json() and see how strings are referenced from there